packages/contracts-core/README.md

```diff
@@ -39,3 +39,34 @@ root
 ├── <a href="./script">script</a>: Scripts for deploying + interacting with contracts
 ├── <a href="./test">test</a>: Test contracts
 </pre>
+
+
+## Running a devnet
+
+<!-- TODO: this doesn't belong here -->
+
+To run a devnet, you can run `make devnet-up` and `make devnet-deploy` from this directory. This will start a local devnet and deploy the contracts to it. RPC endpoints for debugging etc will be availabe at `http://localhost:9001/rpc/[chain_id]`.
+
+By default, the [`PingPongClient.sol`](contracts/client/PingPongClient.sol) is deployed, so you can interact with it with cast. For instance, to send a ping from chain 42 to chain 44:
+
+```bash
+cast send 0x521F44132489CDD54c9ceC8167CfC377CbAEa351 --rpc-url http://localhost:9001/rpc/43 --private-key 0x526db1890baf94e82162f17f25ad769eb7f981272d8d99c527ea1af443c2d0cc "doPing(uint32,address,uint16)" 42 0x521F44132489CDD54c9ceC8167CfC377CbAEa351 44
```

Comment:
To do a single ping-pong round, `42 -> 44 -> 42`:
```suggestion
cast send 0x521F44132489CDD54c9ceC8167CfC377CbAEa351 --rpc-url http://localhost:9001/rpc/42 --private-key 0x526db1890baf94e82162f17f25ad769eb7f981272d8d99c527ea1af443c2d0cc "doPing(uint32,address,uint16)" 44 0x521F44132489CDD54c9ceC8167CfC377CbAEa351 0
```

To do two rounds, `42 -> 44 -> 42 -> 44 -> 42`:
```suggestion
cast send 0x521F44132489CDD54c9ceC8167CfC377CbAEa351 --rpc-url http://localhost:9001/rpc/42 --private-key 0x526db1890baf94e82162f17f25ad769eb7f981272d8d99c527ea1af443c2d0cc "doPing(uint32,address,uint16)" 44 0x521F44132489CDD54c9ceC8167CfC377CbAEa351 1
```